Makefile

```diff
@@ -97,6 +97,10 @@ smb:
 smb-windows:
 	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/smbplugin.exe ./pkg/smbplugin
 
+.PHONY: smb-darwin
+smb-darwin:

```

Comment:
It would be nice to integrate as much of this makefile into csi-release-tools, so that other csi repos can leverage this functionality too.

Reply:
sure, will try to add this into csi-release-tools, let's merge this PR first. thanks.